frontend/pages/title.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/png" href="assets/icon.png"/>
+    <link rel="stylesheet" href="style.css">
+    <title>Tetris</title>
+    <style>
+        
+    </style>
+</head>
+<body>
+
+    <style type="text/css">
+        .snowContainer{width: 100%; height: 100vh; ; overflow: hidden; position: absolute; z-index: -1; top: 0;}
+        .snowExample{position: absolute; top: -450px; opacity: 100; text-align: center;
+        animation: SnowExampleSlide ease-in infinite;
+        -webkit-animation: SnowExampleSlide ease-in infinite;
+        -moz-animation: SnowExampleSlide ease-in infinite;
+        -webkit-animation-fill-mode: forwards;
+        -moz-animation-fill-mode: forwards;
+        animation-fill-mode: forwards;}
+        @keyframes SnowExampleSlide {from {position: absolute; top: 0px;} to {position: absolute; top: 100vh;}}
+        @-webkit-keyframes SnowExampleSlide {from {position: absolute; top: 0px;} to {position: absolute; top: 900px;}}
+        @-moz-keyframes SnowExampleSlide {from {position: absolute; top: 0px;} to {position: absolute; top: 900px;}}
+    </style>
+
+
+
+
+  <div id="blackBar"></div>
+    <h1 id="title_text">TETRIS</h1>
+    <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ&pp=ygUIcmlja3JvbGw%3D">
+        <button id="Start_Button" type="startbutton">Start</button>
```

Comment:
Typically we wouldnt put a button inside a `a` (link) tag.

Usually `a` tags are for linking to other pages. ie you click on the `a` tag and it changes your browser url.
Buttons are instead usually used to activate some javascript function that we have defined in a script. (an example can be seen on this page: https://www.w3schools.com/jsref/event_onclick.asp)